### ISSUE TITLE
Support armadillo subviews

### ISSUE THREAD

**Issue by xhpohanka** [dstahlke/gnuplot-iostream#1]
Hi, thank you for a nice gnuplot wrapper. As you support armadillo datatypes it would be handy to support also their subviews...

**Comment by dstahlke** [comment]
I agree, that would be a good feature (and not hard to add).  Unfortunately I am oversubscribed at the moment, but I'll leave the request open for when I have more time.
